caom2-dm/build.gradle

```diff
@@ -12,7 +12,7 @@ sourceCompatibility = 1.8
 
 group = 'org.opencadc'
 
-version = '2.4'
+version = '2.4.1'
```

Comment:
Pleasedon't change version here. This is not an installable library and the model won't have tertiary versions.

Reply:
My fault. Fixed.